examples/workflows/workspace_workflow.py

```diff
@@ -0,0 +1,35 @@
+from hera.workflows.container_set import ContainerNode, ContainerSet
```

Comment:
@flaviuvadan could you run `make examples` to ensure that the markdown gets created? I think something broke in the CI and it's not being caught for some reason.